crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -366,7 +375,7 @@ where
             }
             Event::SpendableOutputs { outputs } => {
                 tracing::debug!(?outputs, "Spendable outputs");
-                let destination_address = self.wallet.inner().get_unused_address()?;
+                let destination_address = self.wallet.inner().get_new_address()?;
```

Comment:
❓ why changing from `unused_address` to `new_address` here?

Reply:
thanks for pointing this out, seems there was a bit of a mixup here - I'll clean that up together with the other comment